src/storage/src/hummock/state_store.rs

```diff
@@ -194,74 +194,83 @@ impl HummockStorage {
         user_iterator.rewind().await?;
         Ok(HummockStateStoreIter::new(user_iterator))
     }
-}
-
-impl StateStore for HummockStorage {
-    type Iter = HummockStateStoreIter;
-
-    define_state_store_associated_type!();
 
     /// Gets the value of a specified `key`.
     /// The result is based on a snapshot corresponding to the given `epoch`.
     ///
```

Comment:
Could you also please add the doc for `value_meta` here?

Reply:
fixed